client/app/scripts/superdesk-authoring/views/theme-select.html

```diff
@@ -1,19 +1,31 @@
 <div class="dropdown theme-select pull-right" dropdown>
     <button class="dropdown-toggle" dropdown-toggle>
-        <i class="icon-dots-vertical"></i>
+        <i class="icon-chevron-down-thin"></i>
     </button>
-    <div class="dropdown-menu pull-right">
-        <ul>
+    <div class="dropdown-menu pull-right" ng-click="$event.stopPropagation()">
+        <ul class="normal-theme-list">
             <li>
-                <div ng-if="key === 'theme'" class="menu-label" translate>Choose Normal Theme</div>
-                <div ng-if="key === 'proofreadTheme'" class="menu-label" translate>Choose Proofread Theme</div>
+                <i class="theme-status" ng-class="{active: key === 'theme'}"></i>
+                <div class="menu-label" translate>Choose Normal Theme</div>
             </li>
             <li ng-repeat="t in themes">
-                <button ng-click="changeTheme(t)" class="{{t.cssClass}}">
+                <button ng-click="changeTheme('theme', t)" class="{{t.cssClass}}">
                     {{t.label}}
                     <i class="icon-ok" ng-if="theme === t"></i>
                 </button>
             </li>
         </ul>
+        <ul class="proofread-theme-list">
+            <li>
+                <i class="theme-status" ng-class="{active: key === 'proofreadTheme'}"></i>
+                <div class="menu-label" translate>Choose Proofread Theme</div>
+            </li>
+            <li ng-repeat="t in themes">
```

Comment:
Maybe `track by` theme key in both ng-repeat loops... although it's probably not necessary after all, because the theme list does not tend to change and is predefined in the `AuthoringThemesService`, right?